### DISCUSSION TITLE
Addon explanations on YouTube

### DISCUSSION THREAD

**Discussion by DNin01** [ScratchAddons/ScratchAddons#4724] [Ideas]
Basically, some of the more complicated addons would get their own YouTube videos that each explain all of an addon's functionality and how to use it. Probably @Cool-Scratcher would be talking about and showing everything, just like with the update videos. And there would be some way to access these videos from the full screen settings.

**Comment by WorldLanguages** [comment]
This idea has been around for a while, but it never happened 🤔

**Reply by Cool-Scratcher** [reply]
I'll add this to the suggestion doc if it's not already there

I'd be willing to do this

**Comment by Secret-chest** [comment]
> And there would be some way to access these videos from the full screen settings.

Please no autoplay

**Reply by DNin01** [reply]
I wasn't thinking of an embedded video, probably just a `?` or `Watch Video` button.
